calyx-extra/calyx_extra/math_gen/exp.py

```diff
@@ -797,58 +797,3 @@ def build_base_is_e(degree, width, int_width, is_signed) -> Program:
     ]
 
     return builder.program
-
-
-if __name__ == "__main__":
```

Comment:
I could be misreading it, but if we no longer have a "main" entry point and `pyproject.toml` has no `project.scripts` entry, it may be impossible to use this generator from the command line.